ignite/pkg/cosmosclient/cosmosclient.go

```diff
@@ -463,31 +476,6 @@ func (c Client) CreateTx(account cosmosaccount.Account, msgs ...sdktypes.Msg) (T
 	}, nil
 }
 
-// prepareBroadcast performs checks and operations before broadcasting messages
-func (c *Client) prepareBroadcast(ctx context.Context, accountName string, _ []sdktypes.Msg) error {
-	// TODO uncomment after https://github.com/tendermint/spn/issues/363
-	// validate msgs.
-	//  for _, msg := range msgs {
-	//  if err := msg.ValidateBasic(); err != nil {
-	//  return err
-	//  }
-	//  }
-
-	account, err := c.account(accountName)
-	if err != nil {
-		return err
-	}
-
-	// make sure that account has enough balances before broadcasting.
-	if c.useFaucet {
-		if err := c.makeSureAccountHasTokens(ctx, account.Address(c.addressPrefix)); err != nil {
-			return err
-		}
-	}
-
-	return nil
```

Comment:
Note for reviewers: this part is no longer needed since we call `makeSureAccountHasToken` before in `CreateTx` func.